doc/changelog/10-standard-library/12018-master+implb-characterization.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  New lemmas about `implb` in library `Bool`
```

Comment:
Could you please list the added lemmas here? This seems to be the current policy if I'm not mistaken.